docs/guides/main.rst

```diff
@@ -60,6 +60,10 @@ Beets works on Python 3.7 or later.
 If you have `pip`_, just say ``pip install beets`` (or ``pip install --user
 beets`` if you run into permissions problems).
 
+If you have pip but wish to update (uninstall and install the current version),
+run ``python -m pip install –upgrade pip``. Ensure the directory with pip’s scripts
+are added to ``PATH``.
+
```

Comment:
Not sure if we should add this. Handling pip upgrade is kind of out of scope for the beets project. I'd like to keep this already very long guide as short as possible. An alternative idea would be to link to the official pip docs that explain upgrading.

Reply:
We in fact have an FAQ about `$PATH`, which could be worth linking to:
https://beets.readthedocs.io/en/stable/faq.html#say-command-not-found